test/tracing.test.js

```diff
@@ -0,0 +1,28 @@
+const Router = require("../lib/express-promise-router");
+const assert = require("chai").assert;
+
+describe("function name", function () {
+  it("uses the wrapped handler's name in a route handler", function () {
```

Comment:
Can you add a test case to make sure that everything works as expected if the function is actually anonymous? eg the name should be empty?

Reply:
Yeah that seems like a good idea!